### ISSUE TITLE
Better new version item names

### ISSUE THREAD

**Issue by poopman97** [ProtocolSupport/ProtocolSupport#1387]
Can you please add item version numbers for old minecraft versions that don't support that item?
For example when 1.4.7 has a totem of undying instead of it just showing as cobblestone it has the name 1.11 totem of undying like in viaversion

**Comment by Codestech1** [comment]
Good idea, but if you add it should be customizable in config! (And Totem Of Undying is from 1.11, not 1.9 L)

**Comment by poopman97** [comment]
> Good idea, but if you add it should be customizable in config! (And Totem Of Undying is from 1.11, not 1.9 L)

Oh yeah I forgot 💀 I'm going to fix that rn
